### DISCUSSION TITLE
how to setup properly

### DISCUSSION THREAD

**Discussion by dzienisz** [Yue-Yang/ChatGPT-Siri#90] [General]
You need to be on Chinese or English to setup proper shortcut. If you try on other language setting up will fail.

**Comment by Yue-Yang** [comment]
> You need to be on Chinese or English to setup proper shortcut. If you try on other language setting up will fail.

you can use other languages to modify the shortcuts by yourself

**Comment by dzienisz** [comment]
![BE1EED66-C33B-453D-8B0B-C736913694BA_1_105_c](https://user-images.githubusercontent.com/3521624/227878584-c53b4c35-9eeb-4009-a261-beb55ece0276.jpeg)
![EB9CFCF0-7773-4708-9638-A146E2AEB8BD_1_105_c](https://user-images.githubusercontent.com/3521624/227878645-23e87b71-abfe-4a8b-ba2a-6494c166e282.jpeg)
Can you help me with this error?

**Comment by Misterbubble31** [comment]
Same issue

**Comment by Yue-Yang** [comment]
You need to add a payment method at https://platform.openai.com/account/billing/payment-methods to use api

**Reply by dzienisz** [reply]
I have added payment method. It's just it's not working in Polish system language

**Comment by Yue-Yang** [comment]
# choices check this:
https://github.com/Yue-Yang/ChatGPT-Siri/discussions/30
